main.ts

```diff
@@ -15,6 +16,8 @@ async function bootstrap() {
   const document = SwaggerModule.createDocument(app, config);
   SwaggerModule.setup('docs', app, document);
 
+  app.use(logger('[:date[iso]] :method :url :status :response-time ms'));
```

Comment:
@matheo97 are we going to use a logger that stores or logs the information in a file that will be located in a specific directory?? I ask this as I see that we have this logger

Reply:
This logger only displays the logs in the console in a specific format but does not store them anywhere, I added it more as a development tool, but the API will run locally in the user's PC, it will be good to save the logs in files and then if any technical call is needed we can audit them and catch errors easier, should I make this change in this PR?